ports/unix/native_base_class.c

```diff
@@ -76,12 +76,21 @@ STATIC const mp_rom_map_elem_t native_base_class_locals_dict_table[] = {
 };
 STATIC MP_DEFINE_CONST_DICT(native_base_class_locals_dict, native_base_class_locals_dict_table);
 
+STATIC mp_obj_t native_base_class_subscr(mp_obj_t self_in, mp_obj_t index_in, mp_obj_t value) {
+    mp_obj_t attribute_value = mp_load_attr(self_in, MP_QSTR_new_attribute);
+    mp_printf(&mp_plat_print, "native base class subscr .new_attribute set to: ");
+    mp_obj_print_helper(&mp_plat_print, attribute_value, PRINT_REPR);
+    mp_printf(&mp_plat_print, "\n");
+    return attribute_value;
```

Comment:
Is this leftover debugging prints?

Reply:
No, the prints in this class are deliberate. They ensure the test code actually calls it. Otherwise you risk the subclass storing and retrieving the right value without the native class.